### DISCUSSION TITLE
VR Rendering Bug Quest 2

### DISCUSSION THREAD

**Discussion by brainw0rm** [cables-gl/cables_docs#501] [! Welcome - readme first!]
When you add an "imagecompose" op to your scene and trigger it more than once (like creating an animated noise texture) the right eye or even the whole scene wont render on the VR device (oculus quest 2 in my case). This also happens in the VR example patch when you trigger the imagecompose op of the pbr texture more than once. Here is a simple example patch from me: https://cables.gl/p/64005e47469c03e9dd3b86af?s=NTLCB2NeRNhg94jD

**Comment by pandrr** [comment]
try to use the second output port "after eyes"

**Reply by brainw0rm** [reply]
In the simple example from above this seems to do the trick, although when the scene gets a bit more complex the same problem persists. What also seems to be the case is that meshes completely vanish once you use a light in the scene like a spotlight. In the link is a scene which doesnt render the left eye (even with image compose rendered after the eye) and where the mesh disappears completely when plugging in the spotlight (tested on Quest 2): 
https://cables.gl/p/640f39f5ad828812501b6e1a?s=B5QjJ6LCP1sPJlow
